Upgraded market-solidity to 0.0.28

pelsasser opened the pull request:
##### Description
Upgraded `market-solidity` to `0.0.28`

##### Checklist
- [x] Linter status: 100% pass
- [x] Changes don't break existing behavior
- [x] Tests coverage hasn't decreased

##### Affected core subsystem(s)
- Whitelist API
- Tests

##### Testing
Manually tested with local truffle install and works well for all features

pelsasser commented:
okay deployed, this is good to go on my end to be rebased and merged!  LMK your thoughts @eswarasai

eswarasai commented:
@pelsasser -- I've rebased and fixed the conflicts. Everything is working fine except for the fix of #169. Should I hard-code the `baseTokenAddress` of the `WETH` in `actions/deploy.js` while deploying `MARKET Contract`?

pelsasser commented:
That works!

0x0c58e89866dda96911a78dedf069a1848618c185 - is the address of fake WETH
0x01b8de20c76ed06c7e93068a45951c26f70be3db - is the address of fake USD
stable coin.

I think we can also in the guided deploy create a dropdown with only these
two options and then a "custom" option.

On Fri, May 25, 2018 at 3:47 AM, Eswara Sai <notifications@github.com>
wrote:

> @pelsasser <https://github.com/pelsasser> -- I've rebased and fixed the
> conflicts. Everything is working fine except for the fix of #169
> <https://github.com/MARKETProtocol/dApp/issues/169>. Should I hard-code
> the baseTokenAddress of the WETH in actions/deploy.js while deploying MARKET
> Contract?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/MARKETProtocol/dApp/pull/173#issuecomment-392001603>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AOZboWGAD-R562sPuea2TjvIrXws8urjks5t19MwgaJpZM4UMt3g>
> .
>



-- 
Phil Elsasser
CTO and Co-Founder

marketprotocol.io
<http://www.marketprotocol.io>
Join Our Telegram Community <https://t.me/Market_Protocol_Chat>

eswarasai commented:
@pelsasser -- You mean by deploying the contract using `Quick Deploy` using either of the above `baseTokenAddress`? Because I've tried deploying contract via `Simplified Deploy` and still could see that it's `NA` in the `Contract Explorer`. Can you please confirm the same?
